packages/fxa-react/styles/ctas.css

```diff
@@ -47,6 +47,10 @@
     @apply bg-blue-500/40 border-transparent text-white/50 cursor-not-allowed;
   }
 
+  .cta-xl {
```

Comment:
We can remove this from `fxa-content-server`'s styles now. 😎

Reply:
Oh drat 😮 thank you!!